### PR TITLE
Init gradle build

### PR DESCRIPTION
**Pull request by birdstorm**
Initial gradle build for #2

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+			include '**/*.proto'
+        }
+        java {
+            // include self written and generated code
+            srcDirs 'src/main/java', 'target/generated-sources/main/java'            
+        }
+    }
+	test {
+		java {
+			srcDirs 'src/test/java'
```
Comment (ice1000) [inline build.gradle]: This srcDir is added by default.

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,154 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'idea'
```
Comment (ice1000) [inline build.gradle]: Is this really used?

**Comment by birdstorm** [comment]
PTAL @ice1000 thx~

**Comment by birdstorm** [comment]
yeah, I have published the original tikv-client to maven, and I think I should do the same for this raw client.

**Comment by ice1000** [comment]
Cool!

**Pull request merged**